packages/lib/src/classes/EventHandler.ts

```diff
@@ -0,0 +1,64 @@
+import { ClientEvents } from "discord.js"
+import { BetterClient, LogLevel } from "../index.js"
+
+export default class EventHandler {
+	/**
+	 * The name of our event.
+	 */
+	public readonly name: keyof ClientEvents
```

Comment:
It would be cool if this was optional and by default inferred from the file name.